src/command_modules/azure-cli-lab/tests/test_lab.py

```diff
@@ -74,6 +74,14 @@ def body(self):
                      JMESPathCheck('galleryImageReference.publisher', 'MicrosoftWindowsServer')
                  ])
 
+        # List claimable vms
+        self.cmd('lab vm list -g {} --lab-name {} --claimable'
+                 .format(rg, LAB_NAME), checks=[JMESPathCheck('length(@)', 1)])
+
+        # claim a vm
+        self.cmd('lab vm claim -g {} --lab-name {} --name {}'
+                 .format(rg, LAB_NAME, linux_vm_name), checks=[NoneCheck()])
```

Comment:
I would expect something like a vm show command as a follow up to check that some property now says "claimed" (or something).  Otherwise, particularly when individual commands "fail silently" this may appear to work but not actually work.